src/components/ProjectListTable/ProjectListTable.js

```diff
@@ -0,0 +1,24 @@
+import { useTranslation } from 'react-i18next';
+import ProjectListItem from '../ProjectListItem/ProjectListItem'
+import './ProjectListTable.scss'
+
+const ProjectListTable = (props) => {
```

Comment:
This might be a thick question 🤔 but is using an actual html table for this no longer the done thing? It's just I thought it had an accessibility feature that helps link scoped table headings to the table data? But I suspect that will break the responsiveness though 😭 Is there a way to achieve the accessibility benefits without using the actual table element?

Reply:
Yeah so I think using tables vs grid is personal preference. I always default to grid as it has a lot more flexibility when it comes to responsive design (I also feel like I'm coding in the 90s using table :sweat_smile:). There shouldn't really be any massive accessibility downsides to my knowledge aside from perhaps the ordering of speech announcement..which we can re-order in grid if need be.

Reply:
Okay, let's stick with grid in that case 👍